src/wizards/quick_task.xml

```diff
@@ -195,6 +201,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           </xsl:if>
 
           <xsl:if test="(/wizard/params/param[name='auto_start']/value = 1) and (/wizard/params/param[name='start_year']/value != '') and (/wizard/params/param[name='start_month']/value != '') and (/wizard/params/param[name='start_day']/value != '') and (/wizard/params/param[name='start_hour']/value != '') and (/wizard/params/param[name='start_minute']/value != '')">
+            <xsl:variable name="start">
+              <xsl:value-of select="substring(string('100' + /wizard/params/param[name='start_year']/value), 2)"/>
```

Comment:
Same as in modify_task.xsl with the number formatting.

Reply:
Also in 7e96e38.